template-appsettings.json

```diff
@@ -1,4 +1,5 @@
 {
     "prefix": "[]",
+    "ignore_cc_prefix": "nocc_",
```

Comment:
What does "nocc_" mean?

Reply:
I didn't knew where to document it, it's basically prefix to commands that makes them ignore custom commands find. For example: **[]nocc_clear** will execute **clear** no without taking custom commands into consideration.

Reply:
Why should custom commands override the normal ones?

Reply:
Custom commands are checked before standard ones, so doing **[]ccadd ccdelete -Message "..."** will break a bot and it's dangerous when working with Regex because you can accidentally create Regex that will match every command.